source/interprocedural_analyses/taint/test/integration/dictionary.py

```diff
@@ -112,7 +112,7 @@ def forward_comprehension_key_sink(arg):
 
 def lists_of_dictionary_iteration_is_precise():
     list_of_dicts = [
-        {"with_feature": _test_source(), "without_feature": 0} for x in []
```

Comment:
I think this change is a mistake. We recently renamed `__test_source()` to `_test_source()` why are we doing this?

Reply:
Yeah, I thought the same. [See](https://github.com/facebook/pyre-check/pull/458#issuecomment-894413997) but linter wanted these changes:/

Reply:
@PulkitMishra maybe, since the linter complains about things not relevant to the changes proposed in the PR, drop those commits? Or, you could do a `git pull upstream main --rebase` because the source branch of the PR is behind main. Caution: this might make you deal with conflicts.

I believe we can always fix linting issues in a seperate PR.

Also, I am failing to see why git doesn't show conflicts here in the PR 👀

Reply:
Yeah I did not pull before pushing. My bad.

But that aside. Here's the weirder thing. Both black and flake8 want different changes for this line 🤷🏽

Reply:
Rebasing on the latest should fix the issue. The lint was addressed in https://github.com/facebook/pyre-check/commit/eec64a5d115902af81ca54cf1188a7bafed5616f

Reply:
done @r0rshark